src/libs/SidebarUtils.ts

```diff
@@ -385,7 +393,8 @@ function getOptionData({
         } else {
             result.alternateText = lastMessageTextFromReport.length > 0 ? lastMessageText : ReportActionsUtils.getLastVisibleMessage(report.reportID, {}, lastAction)?.lastMessageText;
             if (!result.alternateText) {
-                result.alternateText = Localize.translate(preferredLocale, 'report.noActivityYet');
+                // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+                result.alternateText = ReportUtils.formatReportLastMessageText(getReportBeginningOfChatHistoryMessage(report) || Localize.translateLocal('report.noActivityYet'));
             }
         }
     } else {
```

Comment:
In the else condition we also use beginning of chat message. Can we use `getReportBeginningOfChatHistoryMessage` function there too?